dubbo-common/src/main/java/org/apache/dubbo/common/utils/NetUtils.java

```diff
@@ -102,6 +102,7 @@ public static boolean isAnyHost(String host) {
         return Constants.ANYHOST_VALUE.equals(host);
     }
 
+    // FIXME: should remove this method completely

```

Comment:
If the method is to be removed in the future, I would suggest annotate it with `@Deprecated`

Reply:
it's not been decided yet, so **//FIXME** should be fine for now :)